tests/unit/schema/wrappers/test_method.py

```diff
@@ -123,6 +123,16 @@ def test_method_paged_result_field_no_page_field():
                          )
     assert method.paged_result_field is None
 
+    method = make_method('Foo',
+                         input_message=make_message(name='FooRequest', fields=(
+                             make_field(name='page_token', type=9),  # str
+                         )),
+                         output_message=make_message(name='FooResponse', fields=(
+                             make_field(name='next_page_token', type=9),  # str
+                         ))
+                         )
+    assert method.paged_result_field is None
```

Comment:
Do we add tests in this file for `max_results` and for mapped responses?

Reply:
Mapped responses are treated the same here. Checking for repeated fields should be sufficient since mapped fields are also repeated. Test for `max_results` is now also added.